UEBPrep/app/assets/javascripts/jquery.js

```diff
@@ -1,11 +0,0 @@
-/*****************************************************************************
-*
-* File Name:		jQuery v1.11.3
```

Comment:
Because we're not using any jQuery in our .js files, or JS at all for that matter.. also it looks like we have a $ gem anyway